### PR TITLE
Add --kexec flag to reboot (#1207747)

### PR DESCRIPTION
**Pull request by bcl**
This will be used to reboot the installed system using kexec, skipping
the BIOS/Firmware and bootloader to speed things up.

Related: rhbz#1207747
(cherry picked from commit 79b4c6fde5fd4e89652ee0b1c2d40bf57f014250)

### CODE REVIEW

File: pykickstart/commands/reboot.py [hunk 1]
```diff
@@ -102,3 +102,26 @@ def parse(self, args):
...
+        if self.kexec:
+            retval += " --kexec"
+
+        return retval + "\n"
+
+    def _getParser(self):
+        op = F18_Reboot._getParser(self)
+        op.add_option("--kexec", dest="kexec", action="store_true",
+                      default=False)
+        return op
```
Comment (vpodzime) [inline pykickstart/commands/reboot.py]: If I understand this correctly, we also allow things like ``poweroff --kexec``, ``halt --kexec`` and similar. Is it what we want or should we also override ``parse()`` here and check that ``--kexec`` is only used in combination with ``reboot``?

**Comment by bcl** [comment]
Strictly speaking --kexec only makes sense with reboot. As it is written it is allowed with all the variations. Since this is a backport of an existing feature I'd rather change/discuss that for master, not the backport.

**Comment by vpodzime** [comment]
Works for me.

**Comment by vpodzime** [comment]
ACK

**Pull request merged**